src/main/java/racingcar/domain/InvalidCarNameException.java

```diff
@@ -0,0 +1,14 @@
+package racingcar.domain;
+
+public class InvalidCarNameException extends RuntimeException {
+    private final String message;
+
+    public InvalidCarNameException(String message) {
+        this.message = message;
```

Comment:
[개선요청] RuntimeException 에 이미 message 를 인자로 가지는 생성자를 가지고 있어서  이런경우 아래와 같이 작성하면 됩니다.
```suggestion
      super(message);
```

private final String message; 와 toString() 은 제거 해주세요